docs/application/tizen-studio/release-notes/3-6-release-notes.md

```diff
@@ -0,0 +1,59 @@
+# Tizen Studio 3.6 Release Notes
```

Comment:
@surbhig12 As a new page is added, please guide the author to update the toc_all.md also.

I think @manishtr can refer the update in https://github.com/Samsung/tizen-docs/pull/890.